bjoern/request.c

```diff
@@ -291,8 +301,18 @@ parser_settings = {
 
 void _initialize_request_module()
 {
+    IO = PyImport_ImportModule("io");
+    if (IO == NULL) {
+	    /* PyImport_ImportModule should have exception set already */
+	    return;
+    }
+    STR_bytesio = _Unicode_FromString("BytesIO");
+    STR_write = _Unicode_FromString("write");
+    STR_read = _Unicode_FromString("read");
+    STR_seek = _Unicode_FromString("seek");
+    VALUE_zero = _FromLong(0);
```

Comment:
This should be put into the common module as well. Or simply leave it inline in the `seek` call; it's not as expensive as a string allocation.